include/arch/arm/arch/fastpath/fastpath.h

```diff
@@ -18,6 +18,15 @@ static inline
 void fastpath_call(word_t cptr, word_t r_msgInfo)
 NORETURN;
 
+
+void vm_fault_slowpath(vm_fault_type_t type)
+NORETURN;
+
+static inline
+void fastpath_vm_fault(vm_fault_type_t type)
+NORETURN;
+
+
```

Comment:
these should be guarded by `EXCEPTION_FASTPATH`

(also no need for additional blank lines, one is enough)